src/main/docker/RNApeg.sh

```diff
@@ -23,6 +23,7 @@ while [ ! -z "$1" ]; do
         -b) BAMFILE=$2; shift;;
         -r) REFFLAT=$2; shift;;
         -f) FASTA=$2; shift;;
+        -O) OUTPUT_DIR=$2; shift;;
```

Comment:
Now that I'm thinking about this, how will you set this in CGC? Maybe the output should just default to `dirname($BAMFILE)`.

Reply:
Probably something like 
```
$(inputs.out_dir)
```
???

Reply:
The plan is to now use the default